extensions/resource-deployment/src/ui/notebookWizard/notebookWizard.ts

```diff
@@ -91,8 +92,11 @@ export class NotebookWizard extends WizardBase<NotebookWizard, NotebookWizardPag
 				this.notebookService.backgroundExecuteNotebook(this.wizardInfo.taskName, notebook, 'deploy', this.platformService, env);
 			} else {
 				Object.assign(process.env, env);
-				const title = path.basename(this.notebookService.getNotebookPath(this.wizardInfo.notebook));
-				await this.notebookService.launchNotebookWithContent(title, JSON.stringify(notebook, undefined, 4));
+				const notebookPath = this.notebookService.getNotebookPath(this.wizardInfo.notebook);
+				const ext = path.extname(notebookPath);
+				const title = path.basename(notebookPath, ext);
```

Comment:
Why are you splitting this all up so much? You should just be able to leave the title line the same and modify the launchNotebookWithContent call to add the timestamp

Reply:
I can add the datetimestring within launchNotebookWithContent but I still need to strip out the basename without the extension because the first parameter to launchNotebookWithContent is just a title and not a path. foo.txt-2020020613454 would be weird title instead of foo-2020020613454.

Reply:
ok I will move all logic within launchNotebookWithContent and retain the ext if one was present.

Reply:
moved